documentation/docs/30-capabilities/30-prompts.md

```diff
@@ -6,21 +6,4 @@ This is the list of available prompts provided by the MCP server. Prompts are se
 
 ## svelte-task
 
-This prompt should be used whenever you are asking the model to work on some Svelte-related task. It will instruct the LLM on which documentation sections are available, which tool to invoke, when to invoke it, and how to interpret the result. It will ask you for the description of the task and the returned value will look like this:
```

Comment:
I don't like putting the prompt verbatim in the docs, that's just asking to forget to update it. 😅 If we really want to we could link to the GitHub source code for the prompt.